web/client/components/catalog/editor/AdvancedSettings/WMSDomainAliases.js

```diff
@@ -50,7 +50,7 @@ export default ({
             {toInteger(k) !== 0 &&
             <TooltipButton
                 tooltip={<Message msgId="catalog.domainAliases.removeAliasTooltip" />}
-                tooltipid="add-alias-button"
+                tooltipid="remove-alias-button"
```

Comment:
Dear @youssefallalix , thanks for the contribution. I double check this prop and it seems it is not actually used by the component and it is passed directly to the html [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes)

![image](https://github.com/user-attachments/assets/7352b0ca-74b7-4d4f-a79f-582673fd0037)

so we could simple remove it.